luigi/worker.py

```diff
@@ -268,6 +280,11 @@ class worker(Config):
 
     ping_interval = FloatParameter(default=1.0,
                                    config_path=dict(section='core', name='worker-ping-interval'))
+    task_ping_ratio = IntParameter(default=60,
+                                   config_path=dict(section='core', name='worker-task-ping-ratio'),
```

Comment:
I mean. It's deprecated, we just have it for being able to keep backward compatibility